mlflow/sklearn/__init__.py

```diff
@@ -958,6 +959,22 @@ def patched_fit(original, self, *args, **kwargs):
                     FLAVOR_NAME, class_def, func_name, patched_fit, manage_run=True,
                 )
 
+    def patched_fn_with_autolog_disabled(original, *args, **kwargs):
+        with temporarily_disable_autologging(FLAVOR_NAME):
+            return original(*args, **kwargs)
+
+    from sklearn import model_selection
+    for disable_autolog_func_name in [
+        'cross_validate',
+        'cross_val_predict',
+        'cross_val_score',
+    ]:
+        safe_patch(
+            FLAVOR_NAME, model_selection, disable_autolog_func_name,
+            patched_fn_with_autolog_disabled,
+            manage_run=False
+        )
+
```

Comment:
Can we add tests to ensure autologging is disabled during these `cross_xxx` functions?